cypress/support/metaTestHelper.js

```diff
@@ -84,3 +80,31 @@ export const retrieve404BodyResponse = (url, bodyResponse) => {
     .its('body')
     .should('include', bodyResponse);
 };
+
+export const returnMetadata = () => {
+  cy.window().then(win => {
+    const windowData = win.SIMORGH_DATA.data;
+    const description = windowData.promo.summary;
+    const { language } = windowData.metadata.passport;
+    const title = windowData.promo.headlines.seoHeadline;
+    const { type } = windowData.metadata;
+    const firstPublished = new Date(
+      windowData.metadata.firstPublished,
+    ).toISOString();
+    const lastPublished = new Date(
+      windowData.metadata.lastPublished,
+    ).toISOString();
```

Comment:
```suggestion
    ).toISOString();

```
Nice to separate variable declarations and function calls